ocs_sample_library_preview/OCSClient.py

```diff
@@ -14,9 +14,16 @@
 from .Types import Types
 from .Users import Users
 
+import warnings
+warnings.filterwarnings("default", category=DeprecationWarning, module=__name__)
+
 class OCSClient:
+    warnings.warn(
+        "OCSClient is deprecated as OSIsoft Cloud Services has now been migrated to AVEVA Data Hub, use ADHClient instead",
+        DeprecationWarning
+    )
     """
-    A client that handles communication with OCS
+    A client that handles communication with OCS. NOTE: OCSClient is deprecated as OSIsoft Cloud Services has now been migrated to AVEVA Data Hub, use ADHClient instead.
     """
```

Comment:
rather than just adding the warning and leaving the code as is, can we have the OCSClient create an ADHClient instead? something like def __init__(self, params...): return ADHClient(params...). that way the inner code of this class just points to the ADHclient's code, effectively, and there's not two places where updates need to be managed over time as we decide how long to keep it active. I think if OCSClient still exists, it should be maintained over time to match ADHClient, and this is the easiest way.

Reply:
That is a good suggestion! TIL we can't use __init__ since it's strictly for initializing and can't return anything, but the __new__ method works for this (Not sure if you are already familiar with this concept but if not I found this article good https://www.geeksforgeeks.org/__new__-in-python/).

I made a PR for that and the header renaming fix here [https://github.com/osisoft/sample-ocs-sample_libraries-python/pull/45](url)

Thank you!